sap/latest/support/index.md

```diff
@@ -6,8 +6,5 @@ The following are the supported platforms for Alfresco Content Connector for SAP
 
 | Version | Notes |
 | ------- | ----- |
-| Content Services 7.4.x | |
-| Content Services 7.3.x | |
-| Content Services 7.2.x | |
-| Content Services 7.1.x | |
-| Content Services 7.0.x | |
+| Content Services 23.1 | |
+| Community Edition 23.1 | |
```

Comment:
Please delete this line - the SAP Connector is only supported with ACS Enterprise
```suggestion
| Community Edition 23.1 | |
```